account/generator/account.go

```diff
@@ -25,9 +26,12 @@ func (a *account) toAccountInfo() AccountInfo {
 
 func (a *account) toIdentifiedAccountInfo(id string) IdentifiedAccountInfo {
 	info := a.toAccountInfo()
+	keyUID := sha256.Sum256(crypto.FromECDSAPub(&a.privateKey.PublicKey))
+	keyUIDHex := statusproto.EncodeHex(keyUID[:])
 	return IdentifiedAccountInfo{
 		AccountInfo: info,
 		ID:          id,
+		KeyUID:      keyUIDHex,
```

Comment:
Why not `KeyID`? And what is `id` in this context? Can we add a comment because it gets confusing why we suddenly have two IDs.

Reply:
I agree that it might be `KeyID`, the thing is that the choice of the name has been done already, and it is widely used in many repos
https://github.com/status-im/react-native-status-keycard/search?q=uid&unscoped_q=uid
https://github.com/status-im/keycard-cli/search?q=uid&unscoped_q=uid
https://github.com/status-im/status-keycard/search?q=uid&unscoped_q=uid
https://github.com/status-im/status-keycard-java/search?q=uid&unscoped_q=uid

Even in this repo, although as `keycard-key-uid`:
https://github.com/status-im/status-go/search?q=keycard-key-uid&unscoped_q=keycard-key-uid

The problem now is that we either need to change it to `KeyID`(`key-id`) everywhere or some of repos (almost likely `status-react` and `status-go` atm) will have both and this won't make things easier to maintain and understand, raising the question "What's the difference?" each time you encounter both names.

Reply:
Got it, thanks for the explanation. Let's keep it as it is then and only explain what it is and how it is different from `ID`.